robot.js

```diff
@@ -81,83 +81,129 @@
       const tr = document.createElement('tr');
       table.appendChild(tr);
       let rowHtml = '';
+      let cell = '';
       for (let col = 0; col < cells.length; col++) {
-        const cell = cells[col] === '.' ? '' : cells[col];
-        rowHtml += `<td>${cell}</td>`;
+        if ( cells[col] === '.') {
+           cell = '';
+      } else {
+        cell = cells[col];
       }
-      tr.innerHTML = rowHtml;
```

Comment:
Your indentation is incorrect. You code:

```js
      for (let col = 0; col < cells.length; col++) {
        if ( cells[col] === '.') {
           cell = '';
      } else {
        cell = cells[col];
      }
      //const cell = cells[col] === '.' ? '' : cells[col];
      rowHtml += `<td>${cell}</td>`;
    }
```

Properly formatted:

```js
      for (let col = 0; col < cells.length; col++) {
        if (cells[col] === '.') {
          cell = '';
        } else {
          cell = cells[col];
        }
        //const cell = cells[col] === '.' ? '' : cells[col];
        rowHtml += `<td>${cell}</td>`;
      }
```

Please check [Code Formatting](https://github.com/HackYourFuture/JavaScript/blob/master/fundamentals/code_formatting.md#use-vscode-for-well-formatted-javascript).